README.md

```diff
@@ -14,5 +14,13 @@ You can find the registry of changes in the `CHANGES.md` file.
 
 Every time you want to make a change in this repo, please run the test or generate the proper ones.
 
-To Run unit tests use the following
-  `docker-compose run tests`
+## Developing
+
+```console
+virtualenv venv
+source venv/bin/activate
+pip install -e .
+py.test tests
+docker-compose run test
```

Comment:
```
py.test tests
￼docker-compose run test
```
There is not mandatory to run both lines, they have the same target that is run `py.test tests`. The first uses the `venv` configured and the second first run the docker-compose build and executed a service declared in `docker-compose.yaml`.